File: CS/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraPrinting;
// ...

namespace CommVisibility {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            // Create a report object.
            XtraReport1 report = new XtraReport1();

            // Get the report's printing system.
            PrintingSystem ps = report.PrintingSystem;

            // Hide the Watermark toolbar button, and also the Watermark menu item.
            if(ps.GetCommandVisibility(PrintingSystemCommand.Watermark) != CommandVisibility.None) {
                ps.SetCommandVisibility(PrintingSystemCommand.Watermark, CommandVisibility.None);
            }

            // Show the Document Map toolbar button and menu item.
            ps.SetCommandVisibility(PrintingSystemCommand.DocumentMap, CommandVisibility.All);

            // Make the "Export to Csv" and "Export to Txt" commands visible.
            ps.SetCommandVisibility(new PrintingSystemCommand[] { PrintingSystemCommand.ExportCsv, PrintingSystemCommand.ExportTxt }, CommandVisibility.All);

            // Show the report's preview.
            report.CreateDocument();
            ps.PreviewFormEx.Show();
        }

    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraPrinting
' ...

Namespace WindowsFormsApplication1
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private report As XtraReport1

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            report = New XtraReport1()
            documentViewer1.PrintingSystem = report.PrintingSystem
            ' Get the printing system of the document viewer control.
            Dim ps As PrintingSystemBase = documentViewer1.PrintingSystem

            ' Hide the Watermark toolbar button, and also the Watermark menu item.
            If ps.GetCommandVisibility(PrintingSystemCommand.Watermark) <> CommandVisibility.None Then
                ps.SetCommandVisibility(PrintingSystemCommand.Watermark, CommandVisibility.None)
            End If

            ' Show the Document Map toolbar button and menu item.
            ps.SetCommandVisibility(PrintingSystemCommand.DocumentMap, CommandVisibility.All)

            ' Make the "Export to Csv" and "Export to Txt" commands visible.
            ps.SetCommandVisibility(New PrintingSystemCommand() { PrintingSystemCommand.ExportCsv, PrintingSystemCommand.ExportTxt }, CommandVisibility.All)

            ' Create the document.
            report.CreateDocument()
        End Sub
    End Class
End Namespace
